hooks/php-ast-only.py

```
@@ -0,0 +1,158 @@
+        tokens = command.split()
+
+    program = None
+    for token in tokens:
+        if token in IN_PLACE_PROGRAMS:
+            program = token
+            continue
+        if token == "apply_patch":
+            return "apply_patch"
+        if program is not None and is_in_place_flag(token):
+            return f"{program} in place"
+        if token in REDIRECTS or token == "tee":
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "token"

(S105)